packages/buefy-next/src/utils/InjectedChildMixin.js

```diff
@@ -9,6 +9,22 @@ export const Optional = optional
 export default (parentItemName, flags = 0) => {
     const mixin = {
         inject: { parent: { from: 'b' + parentItemName, default: false } },
+        props: {
+            // if `value` is non-null, it must be unique among all the siblings.
+            // see `uniqueValue`
+            value: {
+                type: String,
+                default: null
+            }
+        },
+        computed: {
+            // parent uses this to identify the child in a collection.
```

Comment:
Is this correct below?
```javascript
            // Parent uses the uniqueValue() method to identify 
            // the child in a collection bound to the `value` prop. 
            // if `value` is non-null, otherwise to `uid` internal field
```

Reply:
I will change it to:
```js
            // `ProviderParentMixin` uses `uniqueValue` computed value to
            // identify the child in its `childItems` collection.
            // so the value must be unique among all the siblings.
            // falls back to the `uid` internal field to ensure uniqueness.
```
Does it make sense?